scripts/release.sh

```diff
@@ -78,7 +111,29 @@ function publish_to_gcs() {
   [[ -n ${TAG} ]] && verbose_gsutil_cp previous/${TAG} $@
 }
 
+function publish_to_azblob() {
+  function verbose_az_cp {
+    local DEST="${RELEASE_AZ_BLOB_URL}/$1/"
+    shift
+    for f in $@
+    do
+      echo "Publishing [$f] to ${DEST}"
+      azcopy copy $f ${DEST}
```

Comment:
Same with @Phlak106's comment for azcopy as well.

Reply:
done!